test/grouping.jl

```diff
@@ -144,14 +152,18 @@ end
 
     gd = groupby(df, :g)
 
-    newvar = :n
+    n_str = "new_column"
+    n_sym = :new_column
+    n_space = "new column"
+
     @test @by(df, :g, n = mean(:i)).n == [2.0, 4.5]
     @test @by(df, :g, n = mean(:i) + mean(:g)).n == [3.0, 6.5]
     @test @by(df, :g, n = first(:t .* string.(:y))).n == ["av", "cy"]
     @test @by(df, :g, n = first(Symbol.(:y, ^(:t)))).n == [:vt, :yt]
     @test @by(df, :g, n = first(Symbol.(:y, ^(:body)))).n == [:vbody, :ybody]
     @test @by(df, :g, body = :i).body == df.i
     @test @by(df, :g, transform = :i).transform == df.i
+    # TODO: Make `t = :n1; (cols(t) = ...,)` work
```

Comment:
Is this a TODO for now or for later? If the latter, is there a way to test the failure?

Reply:
I've decided it isn't a failure. We already have lots of ways to make named tuples with variables as the names. I'm deleting this comment.